p2p/module.go

```diff
@@ -27,8 +26,8 @@ const (
 )
 
 type p2pModule struct {
-	bus       modules.Bus
-	p2pConfig modules.P2PConfig // TODO (olshansky): to remove this since it'll be available via the bus
+	bus    modules.Bus
+	p2pCfg *typesP2P.P2PConfig // TODO (olshansky): to remove this since it'll be available via the bus
```

Comment:
Blocker:

Another example of the casting, this should be `modules.P2PConfig`

Reply:
Maybe addressed with https://github.com/pokt-network/pocket/pull/235#discussion_r993986279